src/store/createStore.ts

```diff
@@ -83,12 +86,16 @@ function isPatch(patchObj: any): patchObj is {id: string; patch: Patch<any, any>
 }
 
 const createStore: StoreFactory = compose<Store<{}, {}, any>, StoreOptions<{}, {}>>({
-	get(this: Store<{}, {}, any>, ids: string[] | string): Promise<{}[]> {
+	get(this: Store<{}, {}, any>, ids: string[] | string): Promise<any> {
```

Comment:
Could we type the return here to `Promise<{} | {}[]>`? That way it provides at least some guarantee about the return type in the implementation.